### PR TITLE
Query activeContractBundles to get embark story id for moving flow. Rename usecase.

### PR DESCRIPTION
**Pull request by hugokallstrom**
### Checklist

- [x] Functionality is covered by an integration test
- [x] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt [hunk 1]
```diff
@@ -109,6 +109,13 @@ class EmbarkActivity : BaseActivity(R.layout.activity_embark) {
         if (offerKeys != null && offerKeys.isNotEmpty()) {
             val offerIds = offerKeys.mapNotNull { model.getFromStore(it) }
             showWebOffer(offerIds)
```
Comment (hugokallstrom) [inline app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt:111]: I just check for the "Offer" passage here. Can we do this in a neater way?

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt:111]: There should be an `ExternalRedirect` here, I believe, that instructs us where to go, and probably also which keys to provide as context to that external redirect location. 🤔

File: app/src/main/java/com/hedvig/app/feature/home/ui/changeaddress/GetAddressChangeStoryIdUseCase.kt [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
 ) {
 
     suspend operator fun invoke(): SelfChangeEligibilityResult {
-        return when (val result = apolloClient.query(SelfChangeEligibilityQuery()).safeQuery()) {
+        return when (val result = apolloClient.query(ActiveContractBundlesQuery()).safeQuery()) {
             is QueryResult.Success ->
-                result.data?.selfChangeEligibility?.embarkStoryId
+                result.data
+                    .activeContractBundles
+                    .firstOrNull()
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/home/ui/changeaddress/GetAddressChangeStoryIdUseCase.kt:20]: I suppose first is going to be fine for now, but long term we likely want this to be able to address any of the contract bundles.

That's a project for when we have UI for handling multiple bundles though. But let's keep it in mind 🙂

**Review by hedvigoscar — APPROVED** [review]
lgtm

**Pull request merged**